schema/manifest_backwards_compatibility_test.go

```diff
@@ -49,10 +49,9 @@ func TestBackwardsCompatibilityManifestList(t *testing.T) {
 		fail         bool
 	}{
 		{
-			digest: "sha256:219f4b61132fe9d09b0ec5c15517be2ca712e4744b0e0cc3be71295b35b2a467",
+			digest: "sha256:ce939cd86094ec89abba1168d55257dfe74781f438916737d5c0905d7afb8803",
 			manifestlist: `{
    "schemaVersion": 2,
-   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
```

Comment:
For this I am a bit confused, if you can not modify the examples of these tests, these original test data is how to get?